CHANGELOG.develop

```diff
@@ -423,6 +424,16 @@ Benner and Paweł Siudak):
 - Renamed =spacemacs/mplist-get= function to =spacemacs/mplist-get-values=,
   renamed =spacemacs/plist-get= function to =spacemacs/mplist-get-value=, and
   refactored the latter function (thanks to Benjamin Reynolds)
+- Fixed pdumper enabled on ~SPC f e R~ no matter what (thanks to Sylvain Benner)
+- Display time spent in =user-config= in home buffer (thanks to Sylvain Benner)
+- Don't call =user-load= unless we are dumping (thanks to Sylvain Benner)
+- Removed dotspacemacs variable =dotspacemacs-import-env-vars-from-shell= (thanks to Sylvain Benner)
+- Removed dotspacemacs variable =dotspacemacs-improt-env-vars-shell-file-name= (thanks to Sylvain Benner)
```

Comment:
The `dotspacemacs-import` variables were added since the last release, so it isn't necessary to add entries for their removal.